packages/server/test/migration/utils/database.ts

```diff
@@ -0,0 +1,73 @@
+import knex, { Knex } from 'knex'
+import schemaInspector from 'knex-schema-inspector'
+import { SchemaInspector } from 'knex-schema-inspector/dist/types/schema-inspector'
+import { Table } from 'knex-schema-inspector/dist/types/table'
+import path from 'path'
+
+export class Inspector {
+  private _knex: Knex
+  private _url: string
+  private _isLite: boolean = false
+  private _inspector: SchemaInspector
+
```

Comment:
Why underscores? We don't usually use them in the codebase

Reply:
Personal preference. Let's add an ESLint rule to enforce it later on. Something like: https://eslint.org/docs/rules/no-underscore-dangle